site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -3382,7 +3386,7 @@ public int compare(String s1, String s2) {
 			
 			context.put("basedOnTemplate",  state.getAttribute(STATE_TEMPLATE_SITE) != null ? Boolean.TRUE:Boolean.FALSE);
 			
-			context.put("requireAuthorizer", ServerConfigurationService.getString("wsetup.requireAuthorizer", "true").equals("true")?Boolean.TRUE:Boolean.FALSE);
+			context.put("requireAuthorizer", ServerConfigurationService.getString(SAK_PROP_REQUIRE_AUTHORIZER, "true").equals("true")?Boolean.TRUE:Boolean.FALSE);
```

Comment:
Yeah, for that property (wsetup.requireAuthorizer) we can, I just did a search replace of the string here so didn't notice the getString verses getBoolean.

However later I use getString instead of getBoolean for the SAK_PROP_EMAIL_AUTHORIZER even though getBoolean would be more appropriate. But the problem is getBoolean returns a primitive boolean rather than the object Boolean and I need to be able to tell when it isn't defined as it has 3 behaviors. I think the API for SCS is wrong for this case but getString is the typical ugly workaround.